forge-gui/res/cardsfolder/upcoming/tivit_seller_of_secrets.txt

```diff
@@ -0,0 +1,13 @@
+Name:Tivit, Seller of Secrets
+ManaCost:3 W U B
+Types:Legendary Creature Sphinx Rogue
+PT:6/6
+K:Flying
+K:Ward:3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigVote | TriggerDescription$ Whenever CARDNAME enters the battlefield or deals combat damage to a player, starting with you, each player votes for evidence or bribery. For each evidence vote, investigate. For each bribery vote, create a Treasure token.

```

Comment:
Use NICKNAME like in the oracle

Reply:
Also add `Council's dilemma —` to the TriggerDesc in line 7 and 8